peachjam/js/components/DocumentContent/index.ts

```diff
@@ -30,18 +30,27 @@ class DocumentContent {
   private searchApp: any;
   private navOffCanvas: OffCanvas | undefined;
   private enchrichmentsManager: EnrichmentsManager | null;
+  private originalDocCloned: Node | undefined;
+  private tocController: HTMLElement| null;
+  private documentElement: Element | null;
   constructor (root: HTMLElement) {
     this.root = root;
     this.navOffCanvas = undefined;
     this.enchrichmentsManager = null;
+    this.documentElement = this.root.querySelector('[data-document-element]');
+    this.originalDocCloned = this.documentElement?.cloneNode(true);
+    this.tocController = this.setupTocForTab();
+
+    if (root.dataset.frbrUriSubtype === 'book' && root.getAttribute('data-display-type') === 'akn') {
```

Comment:
Please delegate this to an attribute on `root` -- that makes it easier for us to determine in python to en/disable this behaviour, and means this component doesn't need to know about subtypes etc.

Reply:
eg. `data-toc-show-active-item-only` or something. If that's present, then this behaviour is turned on.

Reply:
done